src/makeRobotAccountant.js

```diff
@@ -6,7 +6,17 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
-}
+  let count = 3;
```

Comment:
A nice solution only for this task. Better to use the initial value of 0, for better understanding. But to pass these tests, is okay

P.S. Starting from a value of 0 it looks more natural =))